readme.md

```diff
@@ -502,6 +502,7 @@
 
 ## Content Management Systems
 
+- [Sitecore](https://github.com/MartinMiles/Awesome-Sitecore)
```

Comment:
Please move your entry to the bottom of the category.

Requirement:
> Your entry should be added at the bottom of the appropriate category.

Reply:
Done